CartoDB/cartodb
# icons on custom legends

close https://github.com/CartoDB/cartodb/issues/10726

**steps to test this:**

- create a map, add a custom legend, and pick an icon in any of the items
- check the icon (and the color) changes both in Builder and the map

![screen shot 2017-01-10 at 14 59 41](https://cloud.githubusercontent.com/assets/36676/21860269/3a4e8722-d830-11e6-9184-8539b923ac73.png)

this needs to be reviewed and merged first:

- [x] cartodb.js support https://github.com/CartoDB/cartodb.js/pull/1545
